truffleruby/src/main/java/org/truffleruby/core/CoreLibrary.java

```diff
@@ -789,6 +789,7 @@ private void initializeConstants() {
         Layouts.MODULE.getFields(rubiniusFFIModule).setConstant(context, node, "TYPE_VARARGS", RubiniusTypes.TYPE_VARARGS);
 
         Layouts.MODULE.getFields(objectClass).setConstant(context, node, "RUBY_VERSION", frozenUSASCIIString(RubyLanguage.RUBY_VERSION));
+        Layouts.MODULE.getFields(objectClass).setConstant(context, node, "TRUFFLERUBY_VERSION", frozenUSASCIIString(RubyLanguage.getTruffleRubyVersion()));
```

Comment:
Both are fine to me, the advantage of `TRUFFLERUBY_VERSION` is that's easier to check, even if `defined?` is buggy (as we used to have) and it's not the "Truffle version" but the "TruffleRuby version".